geode-protobuf/src/main/proto/authentication_API.proto

```diff
@@ -17,8 +17,7 @@ syntax = "proto3";
 package org.apache.geode.internal.protocol.protobuf;
 
 message SimpleAuthenticationRequest {
-    string username = 1;
-    string password = 2;
+    map<string,string> credentials = 1;
 }
 
 message SimpleAuthenticationResponse {
```

Comment:
This is response is not specifically used for simple authentication any more. Maybe we just have `PropertyBasedAuthenticationResponse` or the much simpler `AuthenticationResponse`

Reply:
AuthenticationResponse it is